### PR TITLE
Fix fetch gather

### PR DESCRIPTION
**Pull request by lpasquali**

### CODE REVIEW

File: docker/ckan/ckan-entrypoint.sh [hunk 1]
```diff
@@ -57,6 +57,9 @@ while ! pg_isready -h $PG_HOST -U ckan; do
   sleep 1;
 done
 
+if [ -f "/tmp/ckan.config.done" ]; then
+  rm /tmp/ckan.config.done
+fi
```
Comment (etj) [inline docker/ckan/ckan-entrypoint.sh]: What is this needed for?

File: docker/ckan/Dockerfile [hunk 2]
```diff
@@ -111,19 +111,22 @@ RUN mkdir -p /var/log/supervisor /var/run/supervisor /etc/supervisor/conf.d
...
-RUN chmod 0644 /etc/cron.d/crontab-ckan
+# RUN chmod 0644 /etc/cron.d/crontab-ckan
 
 # Apply cron job
-RUN crontab -u ckan /etc/cron.d/crontab-ckan
+# RUN crontab -u ckan /etc/cron.d/crontab-ckan
 
 # Create the log file to be able to run tail
-RUN touch /var/log/cron.log
+# RUN touch /var/log/cron.log
```
Comment (etj) [inline docker/ckan/Dockerfile]: I guess that all the cron commented out stuff can be remoevd for good

**Review by etj — CHANGES_REQUESTED** [review]
Some minor cleanup requested, then it's ready to go!

File: docker/ckan/ckan-entrypoint.sh (hunk 1 above)
Comment (lpasquali) [reply docker/ckan/ckan-entrypoint.sh]: removed, thanks

File: docker/ckan/Dockerfile (hunk 2 above)
Comment (lpasquali) [reply docker/ckan/Dockerfile]: done

**Pull request merged**